FeedSX/screens/UniversalFeed/index.tsx

```diff
@@ -215,29 +334,41 @@ const UniversalFeed = () => {
               }}
               // footer props
               footerProps={{
-                isLiked: item.isLiked,
-                isSaved: item.isSaved,
-                likesCount: item.likesCount,
-                commentsCount: item.commentsCount,
+                isLiked: item?.isLiked,
+                isSaved: item?.isSaved,
+                likesCount: item?.likesCount,
+                commentsCount: item?.commentsCount,
                 showBookMarkIcon: true,
                 showShareIcon: true,
                 likeIconButton: {
                   onTap: () => {
-                    postLikeHandler(item.id);
+                    postLikeHandler(item?.id);
                   },
                 },
                 saveButton: {
                   onTap: () => {
-                    savePostHandler(item.id);
+                    savePostHandler(item?.id);
                   },
                 },
                 likeTextButton: {
-                  onTap: () => NavigationService.navigate('LikesList', item.id),
+                  onTap: () => {
+                    dispatch(postLikesClear() as any);
+                    NavigationService.navigate(LIKES_LIST, item?.id);
+                  },
+                },
+              }}
+              mediaProps={{
+                attachments: item?.attachments ? item.attachments : [],
+                videoProps: {videoUrl: '', showControls: true},
+                carouselProps: {
+                  attachments: item?.attachments ? item.attachments : [],
+                  videoItem: {videoUrl: '', showControls: true},
                 },
               }}
             />
           )}
-          estimatedItemSize={5}
+          estimatedItemSize={500}
```

Comment:
Any specific reason to make it 500?

Reply:
Yes, actually the estimatedItemSize defines the estimated height of one item of the list so thats so made it 500 , guessing one post height won't be more than 500px